xonsh/history.py

```diff
@@ -110,7 +110,8 @@ def files(self, only_unlocked=False):
         """
         _ = self  # this could be a function but is intimate to this class
         # pylint: disable=no-member
-        xdd = os.path.abspath(builtins.__xonsh_env__.get('XONSH_DATA_DIR'))
+        xdd = os.path.expanduser(builtins.__xonsh_env__.get('XONSH_DATA_DIR'))
+        xdd = os.path.abspath(xdd)
```

Comment:
Thanks. I'm not very familiar with the code and wasn't able to find the correct object to use the built_in on (`xonsh` isn't defined in this scope, `self` doesn't have a `built_ins` member). More hints appreciated.